<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons/main.qrc">
    <normaloff>:/surfer.png</normaloff>:/surfer.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="m_tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_contentsTab">
      <attribute name="title">
       <string>Contents</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="m_imagesCheckBox">
         <property name="text">
          <string>Images</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_javascriptCheckBox">
         <property name="text">
          <string>Javascript scenarios</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_javaCheckBox">
         <property name="text">
          <string>Java applets</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_pluginsCheckBox">
         <property name="text">
          <string>Plugins (flash, etc)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_fontsTab">
      <attribute name="title">
       <string>Fonts</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QFontComboBox" name="m_serifFontComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentFont">
          <font>
           <family>Liberation Serif</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QFontComboBox" name="m_monoFontComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="fontFilters">
          <set>QFontComboBox::MonospacedFonts</set>
         </property>
         <property name="currentFont">
          <font>
           <family>Liberation Mono</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="m_serifFontLabel">
         <property name="text">
          <string>Serif Font</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="m_sansFontLabel">
         <property name="text">
          <string>Sans Font</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFontComboBox" name="m_sansFontComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="m_standardFontSpinBox">
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>25</number>
         </property>
         <property name="value">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="m_monoFontLabel">
         <property name="text">
          <string>Monospace Font</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QSpinBox" name="m_monoFontSpinBox">
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>25</number>
         </property>
         <property name="value">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="3">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_historyTab">
      <attribute name="title">
       <string>History</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="m_privateBrowsingCheckBox">
         <property name="text">
          <string>Private Browsing (visited pages will be not written to history)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="m_historyExpirationLabel">
           <property name="text">
            <string>Store each history item during</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_historyExpirationComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>2</number>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>1 day</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3 days</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1 week</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2 weeks</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1 month</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3 month</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6 month</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1 year</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_diskTab">
      <attribute name="title">
       <string>Disk</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="m_downloadCheckBox">
           <property name="text">
            <string>Download everything to</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="m_downloadLineEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_downloadToolButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string notr="true">...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="m_cacheSizeLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Disk cache size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="m_cacheSizeSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="suffix">
            <string> Mb</string>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="m_defaultsButton">
       <property name="text">
        <string>Return to defaults</string>
       </property>
       <property name="icon">
        <iconset resource="../icons/main.qrc">
         <normaloff>:/defaults.png</normaloff>:/defaults.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="m_horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>144</width>
         <height>22</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="../icons/main.qrc">
         <normaloff>:/ok.png</normaloff>:/ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset resource="../icons/main.qrc">
         <normaloff>:/apply.png</normaloff>:/apply.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../icons/main.qrc">
         <normaloff>:/cancel.png</normaloff>:/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/main.qrc"/>
 </resources>
 <connections/>
</ui>
